docs/Users_Guide/appendixH.rst

```diff
@@ -0,0 +1,63 @@
+UGRID convention element name: dim_vert
+Coordinate variable name for the vertical coordinate, if present (optional)
+UGRID convention element name: vert_face
+
+Unstructured Grid Configuration Elements
+========================================
+
+In the PointStat and GridStat config files, a user can control aspects of the UGRID capability.
+
+**ugrid_dataset [optional]**
+If the UGRID dataset is supported by MET, then this item is set to the string identifying the UGRID. Currently supported ugrid_dataset options include “lfric”, and “mpas”. If this item is not set, the user must provide a separate UGRID configuration file on the command line using the **-ugrid_config** command line argument.
+
+**ugrid_max_distance_km [default: 0 km]**
+For PointStat, this is the distance from each UGRID cell center that PointStat will search outward to collect observations to use for verification. The default is 0 km, which by default will use the closest observation to the UGRID cell for verification. For GridStat, this is the distance from each UGRID forecast cell center to search for observation grid cells to include when interpolating to the UGRID forecast cell locations. Currently, only the nearest point observation or gridded observation cell is used.
+
+**ugrid_coordinates_file [optional]**
+The absolute path to the ancillary NetCDF file that describes the UGRID topology. This file is checked for the required UGRID metadata first, and the data file is checked second and takes precedence over this file if the same elements are found in both files.
+
+**ugrid_metadata_map [required if not using a supported UGRID]**
+The mapping dictionary which allows a user to specify the variable names of the required UGRID topology elements that are required by MET. For "lfric" and "mpas", **ugrid_metadata_map** comes pre-configured with MET and a user can simply set **ugrid_dataset**. Otherwise, the user must create a separate UGRID configuration file containing **ugrid_metadata_map** and provide it on the command line using the **-ugrid_config** command line argument.
+
+Unstructured Grid Limitations
+=============================
+
+We anticipate expanding the UGRID capabilities in MET in the near future. Until then, users are encouraged to be mindful of the following limitations:
+
+1. In GridStat, there is no support for verifying directly on a UGRID. The UGRID (either forecast, obs, or both) must be regridded to a structured verification grid prior to verification being performed. It is recommended that the user define a custom verification grid tailored to their UGRID characteristics, being mindful that the regridding can be slow for large verification grids.
+
+2. Data at cell edges and nodes are currently not supported, only those variables which have data at the cell centers are supported. Users should note in particular that wind components that are typically derived using data at cell edges are currently unsupported.
```

Comment:
See comment earlier on "nodes", which leads to me to suggest removing reference to nodes from this comment, since its meaning may vary depending on conventions.